tests/test_integration/test_loaders/test_resource_loaders.py

```diff
@@ -719,6 +719,7 @@ def test_load_workflow_without_defaults_not_redeployed(self, toolkit_client: Too
     parameters:
       transformation:
         externalId: some_transformation
+    retries: null
```

Comment:
By modifying the old test case we lose coverage on what what was previously tested, shouldn't we test both missing property and explicit null property?

Reply:
Note that there are two tasks. The case when retries is not set is still covered